From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>
Bug: https://github.com/haskell-nix/hnix-store/pull/167

This is needed for the tests to compile
Without the import, `max_live_bytes` and `getRTSStats` are undefined

--- hnix-store-core-0.5.0.0/tests/NarFormat.hs
+++ hnix-store-core-0.5.0.0-new/tests/NarFormat.hs
@@ -48,6 +48,9 @@
 import qualified System.Nix.Internal.Nar.Streamer as Nar
 import           System.Nix.Nar
 
+#ifdef BOUNDED_MEMORY
+import GHC.Stats
+#endif
 
 
 withBytesAsHandle :: BSLC.ByteString -> (IO.Handle -> IO a) -> IO a
